service/src/main/kotlin/xyz/lebkuchenfm/domain/xsounds/XSoundsService.kt

```diff
@@ -25,9 +25,6 @@ class XSoundsService(private val repository: XSoundsRepository, private val file
     }
 
     suspend fun markAsPlayed(soundName: String) {
-        val xSound = this.getByName(soundName)
-        xSound?.let {
-            repository.incrementPlayCount(it)
-        }
+        repository.incrementPlayCount(soundName)
```

Comment:
Also please add log that something went wrong:
```suggestion
        repository.incrementPlayCount(soundName)?.also { 
            logger.error { "Could not increment play count for sound $soundName" }
        }
```